SwiftGit2Tests/RepositorySpec.swift

```diff
@@ -641,8 +641,65 @@ class RepositorySpec: QuickSpec {
 				expect(commitMessages).to(equal(expectedMessages))
 			}
 		}
+
+		describe("Repository.status	") {
+			it("Should return accurate status") {
+				// Repo with no status
+				let repo = Fixtures.mantleRepository
+				let branch = repo.localBranch(named: "master").value!
+				expect(repo.checkout(branch, strategy: CheckoutStrategy.None)).to(haveSucceeded())
+
+				let status = repo.status()
+
+				expect(status.value?.count).to(equal(0))
```

Comment:
This `expect` and the one below are really two different tests, so they belong in separate `it`s.